content/blog/2021-12-15-Feuchtigkeit-Teutoburger-Wald.md

```diff
@@ -0,0 +1,60 @@
+topic: community, project
+images:
+ - imgname: teuto_wald2.jpg
+ - imgname: teuto_team.jpg # Dieses Bild sollte im Verzeichnis static/blog existieren
+ - imgname: teuto_ranger_sensor.jpg
+
+og_image: teuto_wald2.jpg # Der Dateiname vom Titelbild das kann das selbe sein wie unter images und sollte auch unter static/blog liegen
+og_description:  Ansicht Teutoburger Wald mit abgestorbenen Fichten. # Der alt Text zum Titelbild
+
+# Tags werden benutzt um die twitter card für den artikel mit #Tags zu versorgen und um in Suchmaschinen gefunden zu werden
+tags:
+- Opendata
+- Citizen Science
+- Umweltschutz
+- Gesellschaft
+
+draft: false # So lassen sonst wird der Artikel nicht veröffentlicht
+---
+
+# Bodenfeuchtigkeit sichtbar machen: Innovationen für den Teutoburger Wald
+
+## Klimawandel im Teutoburger Wald 
+
+Auch in der Region Ostwestfalen-Lippe ist der Klimawandel zu beobachten. Insbesondere der Teutoburger Wald hat sich in den letzten Jahren stark verändert. Windbruch und trockene Sommer haben u.a. die Vermehrung des Borkenkäfers begünstigt, was Krankheiten und Absterben des Waldes zur Folge hatte.
+ 
+Die Feuchtigkeit im Boden ist einer von mehreren Indikatoren zur Beschreibung des Zustands des Waldes. Die Bodenfeuchtigkeit variiert im Jahresverlauf, dennoch gibt es aufgrund zunehmender Temperaturen und nach zwei Hitzesommern 2018 und 2019 die Tendenz dazu, dass der Waldboden zu trocken ist. Um diese Entwicklung den Bürgerinnen und Bürgern der Region objektiv näher zu bringen und um u.U. auch geeignete Tools zur Unterstützung einer optimalen Waldbewirtschaftung entwickeln zu können, bietet es sich an, Daten zum Waldzustand zu sammeln und als Open Data frei zur Verfügung zu stellen.
+
+## Bodenfeuchtigkeit messen und darstellen
+
+Code for Bielefeld, eine Gruppe von IT-affinen Bielefelder:innen, die sich ehrenamtlich für digitale Tools in Bielefeld engagieren, arbeitet seit September an einer Umsetzung, um die Feuchtigkeit des Waldbodens in einer online zugänglichen Karte grafisch darzustellen. Dadurch soll die Klimaentwicklung transparenter gemacht und objektive Daten zur Klimaänderung erhoben werden. 
+
+Zurzeit sind 8 Aktive dabei, die richtigen Sensoren auszuwählen, um Bodenfeuchtigkeit zu messen. Ein erster Prototyp sendet bereits Daten aus einem Garten heraus. Die Weitergabe der Daten erfolgt mittels LoRaWAN, einer Funktechnologie, die es schafft, sehr energiesparsam kleine Datenmengen über weite Strecken zu verschicken. Das ermöglicht, dass die Sensoren Batteriebetrieben sind und auf einer weiten Fläche verteilt stehen können. Batterielaufzeiten von mehreren Jahren sind mit dieser Technologie keine Seltenheit. Zur Weitergabe der Daten mittels LoRaWAN wird die Infrastruktur der The Things Network-Community verwendet. 
+
+![Team](https://github.com/okfde/codefor.de/blob/main/static/blog/teuto_team.JPG)
```

Comment:
Deine Bilder müssen ins Verzeichnis `static/blog/feuchtigkeit-teutoburger-wald/`. dann kannst du einfach 
```suggestion
![Team](teuto_team.JPG)
```

Reply:
Hatte mich an der Anleitung orientiert. Kann die Bilder aber gerne in das Verzeichnis packen.

Reply:
Oh dann solte ich mal die Anleitung anpassen :)